frontend/src/app/login/callback/page.tsx

```
@@ -0,0 +1,60 @@
+  useEffect(() => {
+    console.log('In LoginCallback useEffect', searchParams);
+    const redirectionRoute = searchParams?.get('route') || 'home';
+
+    // Todo: Need to evaluate this piece of code
+    const nextRoute =
+      {
+        home: '/',
+        login: '/login',
+      }[redirectionRoute] || '/';
+    const error = searchParams?.get('error');
+    if (!!error) {
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)